hw/ip/rstmgr/rstmgr.core

```diff
@@ -18,12 +18,12 @@ filesets:
       - lowrisc:prim:sparse_fsm
       - "fileset_ip     ? (lowrisc:ip:rstmgr_pkg)"
       - "fileset_top    ? (lowrisc:systems:rstmgr_pkg)"
+      - lowrisc:ip:rstmgr_cnsty_chk
```

Comment:
@matutem might need your help understanding why this one is in depends and not files.